oggm/cfg.py

```diff
@@ -253,6 +249,17 @@ def initialize(file=None):
     if cp['rgi_dir'] == '~':
         cp['rgi_dir'] = os.path.join(homedir, 'OGGM_data', 'rgi')
 
+    # Setup Download-Cache-Dir
+    if os.environ.get('OGGM_DOWNLOAD_CACHE') is not None:
+        cp['dl_cache_dir'] = os.environ.get('OGGM_DOWNLOAD_CACHE')
+    PATHS['dl_cache_dir'] = cp['dl_cache_dir']
+    if PATHS['dl_cache_dir'] and not os.path.exists(PATHS['dl_cache_dir']):
+        os.makedirs(PATHS['dl_cache_dir'])
```

Comment:
The download functions have the same kind of check, and they keep working like they do now if no cache dir is set.